ios/RNShopify.m

```diff
@@ -0,0 +1,133 @@
+
+#import "RNShopify.h"
+#import "Buy.h"
+
+@implementation RNShopify {
+  RCTPromiseResolveBlock _resolve;
+  RCTPromiseRejectBlock _reject;
+}
+
+- (dispatch_queue_t)methodQueue
+{
+    return dispatch_get_main_queue();
+}
+RCT_EXPORT_MODULE()
+
+RCT_EXPORT_METHOD(initialize:(NSString *)domain key:(NSString *)key)
+{
+
+  self.client = [[BUYClient alloc] initWithShopDomain:domain
+                                                     apiKey:key
+                                                      appId:@"8"];
```

Comment:
It's predefined by Shopify's Mobile Buy SDK. It needs to be included in the client but it's always "8".